config/initializers/string.rb

```diff
@@ -11,6 +11,11 @@ def normalize
     mb_chars.normalize(:kd).gsub(/[^\x00-\x7F]/n, "")
   end
 
+  # Temporary method to rid a string of pesky characters that might annoy Power BI.
+  def vanilla
+    normalize.gsub(/[^a-z0-9._\- ]/i, "").to_s
+  end
```

Comment:
As noted in PR description, this is smelly. I should figure out how to remove it but need to prioritize other work this week.

It's used many places because Power BI rejects certain symbols, BUT we don't want to remove everything or the string will look ugly to humans using the interface (top level `ResponsesMymissionwithsymbols` vs. `Responses: My mission - with symbols and spaces`)

Reply:
Yeah the way to do this later I think will be decorators. Fine for now as long as you have a note somewhere to get rid of it later.